store/auth_test.go

```diff
@@ -117,6 +125,45 @@ func (s *authTestSuite) TestRequestStoreTokenInvalidLogin(c *C) {
 	c.Assert(err, Equals, ErrInvalidCredentials)
 }
 
+func (s *authTestSuite) TestRequestPackageAccessMacaroon(c *C) {
+	mockServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		io.WriteString(w, mockStoreReturnMacaroon)
+	}))
+	c.Assert(mockServer, NotNil)
```

Comment:
No need for these. If NewServer returns nil the httptest package is severely broken, and the next line will crash on it.